ush/test/config/bufr_script4backend_atms.yaml

```diff
@@ -0,0 +1,52 @@
+time window:
+  begin: "2018-04-14T21:00:00Z"
+  end: "2023-12-15T03:00:00Z"
+
+observations:
+  - obs space:
+      name: "atms_npp"
+      simulated variables: ["brightnessTemperature"]
+      obsdatain:
+        engine:
+          type: script
+          script file: "bufr_atms.py"
```

Comment:
Should the script file have "./" in front of it? More of a formatting question than anything. Either should work.

Reply:
added